src/negotiation/order.ts

```diff
@@ -1,19 +1,30 @@
 import { BigNumber } from "bignumber.js";
 import { Token } from "../tokens/tokens";
 
+/**
+ * The order of a trade as created by the maker through the {@link MakerNegotiator}.
+ */
 export interface Order {
   id: string;
   validUntil: number;
   bid: OrderAsset;
   ask: OrderAsset;
 }
 
+/**
+ * Representation of the bid or ask of an {@link Order}.
```

Comment:
Is it possible to documention the individual properties? e.g. say that nominal means "Ether" for Ethereum and "Bitcoin" for Bitcoin

Reply:
Not exactly sure what you mean, please add a suggestion for clarification.

Reply:
I'm opting for adding a comment for `nominalAmount` for now (tested it, that works fine), would add more in the next iteration. `ledger` and `asset` are pretty self explanatory, even though we should document how exactly the string should look. (But that probably makes the documentation outdated soon.)